README.md

```diff
@@ -59,6 +59,14 @@ Module | Platform | Sample | Release Notes
 [azure/app-configuration](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/appconfiguration/app-configuration)<br/>[![npm](https://img.shields.io/npm/v/@azure/app-configuration.svg?color=blue)](https://www.npmjs.com/package/@azure/app-configuration) | JavaScript | [Sample](https://github.com/Azure/azure-sdk-for-js/tree/main/sdk/appconfiguration/app-configuration/samples) | [Release Notes](https://github.com/Azure/azure-sdk-for-js/blob/main/sdk/appconfiguration/app-configuration/CHANGELOG.md)
 [azure-appconfiguration](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/appconfiguration/azure-appconfiguration)<br/>[![pypi](https://img.shields.io/pypi/v/azure-appconfiguration.svg?color=blue)](https://pypi.org/project/azure-appconfiguration/) | Python | [Sample](https://github.com/Azure/azure-sdk-for-python/tree/main/sdk/appconfiguration/azure-appconfiguration/samples) | [Release Notes](https://github.com/Azure/azure-sdk-for-python/blob/main/sdk/appconfiguration/azure-appconfiguration/CHANGELOG.md)
 
+## Kubernetes Integration
```

Comment:
We've been using `Azure Kubernetes Service` in other documents, would it be better to use `Azure Kubernetes Service` here as well?

Reply:
We usually use `Azure Kubernetes Service` in Azure learning doc because the target audience is users of Azure. But our k8s provider is not only restricted to AKS, so use `Kubernetes ` is appropriate here.